odoo/orm/decorators.py

```diff
@@ -295,7 +295,20 @@ def method(self, args):
     """
     if method.__name__ == 'create':
         return model_create_multi(method)  # type: ignore
-    method._api = 'model'  # type: ignore
+    if not hasattr(method, '_api'):
+        method._api = 'model'  # type: ignore
+    return method
+
+
+def private(method: C) -> C:
+    """ Decorate a record-style method to indicate that the method cannot be
+        called using RPC.
+
+            @api.private
+            def method(self, args):
+                ...
+    """
+    method._api = 'private'  # type: ignore
```

Comment:
Doesn't it mean that it isn't possible to have @private + @model ?

Reply:
You can decorate using both and "private" will prevail. Model would be there just for documentation.
From the API point of view, when it is private, then model does not make sense.

Reply:
Ho I missed the fact that you changed api.model to ignore it if `method._api` is already set.
It is worth a comment there IMO.